app.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
+
+// Global Variables
+var images = [
+  new Image('img/bag.jpg', 'bag'),
+  new Image('img/banana.jpg', 'banana'),
+  new Image('img/bathroom.jpg', 'bathroom'),
+  new Image('img/boots.jpg', 'boots'),
+  new Image('img/breakfast.jpg', 'breakfast'),
+  new Image('img/bubblegum.jpg', 'bubblegum'),
+  new Image('img/chair.jpg', 'chair'),
+  new Image('img/cthulhu.jpg', 'cthulhu'),
+  new Image('img/dog-duck.jpg', 'dog-duck'),
+  new Image('img/dragon.jpg', 'dragon'),
+  new Image('img/pen.jpg', 'pen'),
+  new Image('img/pet-sweep.jpg', 'pet-sweep'),
+  new Image('img/scissors.jpg', 'scissors'),
+  new Image('img/shark.jpg', 'sharks'),
+  new Image('img/sweep.png', 'sweep'),
+  new Image('img/tauntaun.jpg', 'tauntaun'),
+  new Image('img/unicorn.jpg', 'unicorn'),
+  new Image('img/usb.gif', 'usb'),
+  new Image('img/water-can.jpg', 'water-can'),
+  new Image('img/wine-glass.jpg', 'wine-glass')
+];
+
+console.log('---------List of Images---------');
+console.dir(images);
+
+var lastSet = [];
+console.log('---last Set ---');
+console.log(lastSet);
+var currentSet = [];
+console.log('----current Set----');
+console.log(currentSet);
+
+var sumbitions = 0;
+
+// Form Parent Node
+var formEl = document.getElementById('option-set');
+
+// simple image node creator
+function createImage(url, alt, parentNode) {
+  var element = document.createElement('img');
+  element.setAttribute('src', url);
+  element.setAttribute('alt', alt);
+  // console.log(element);
+  //give the Child to the Dom
+  parentNode.appendChild(element);
+  return element;
+}
```

Comment:
I want a semi colon to end function declarations.  You don't technically need them, but it is common practice